### ISSUE TITLE
DateTime formatting replaces forward slash '/' with hyphen '-'

### ISSUE THREAD

**Issue by vic10us** [scriptcs/scriptcs#1155]
Here is an example output from scriptcs prompt:

```
> Console.WriteLine("{0:MM/dd/yyyy}", DateTime.Now);
04-07-2016
```

**Comment by filipw** [comment]
which scriptcs version are you on?
also, is this Windows or Mono?

FWIW, works on my machine
<img width="392" alt="screenshot 2016-04-07 18 26 40" src="https://cloud.githubusercontent.com/assets/1710369/14368838/4818c8c2-fcee-11e5-8439-cc470b0ff3c8.png">

**Comment by filipw** [comment]
I believe this was fixed with #1216
